src/components/ProfilePage/AddressCard/AddressCard.tsx

```diff
@@ -1,44 +1,60 @@
-import { useMenuStore } from '@/stores/menu';
-import ContentCopyIcon from '@mui/icons-material/ContentCopy';
-import OpenInNewIcon from '@mui/icons-material/OpenInNew';
+import { useAccount } from '@lifi/wallet-management';
+import KeyboardArrowDownIcon from '@mui/icons-material/KeyboardArrowDown';
 import { useTheme } from '@mui/material';
 import Image from 'next/image';
+import { useState } from 'react';
 import { useTranslation } from 'react-i18next';
 import useImageStatus from 'src/hooks/useImageStatus';
 import { useMercleNft } from 'src/hooks/useMercleNft';
+import { useMenuStore } from 'src/stores/menu';
 import { getAddressLabel } from 'src/utils/getAddressLabel';
 import type { Address } from 'viem';
 import { useEnsName } from 'wagmi';
 import { mainnet } from 'wagmi/chains';
-import { NoSelectTypography } from '../ProfilePage.style';
+import { AddressMenu } from '../AddressMenu/AddressMenu';
 import {
+  AddressBox,
   AddressBoxContainer,
-  AddressDisplayBox,
+  AddressButton,
+  AddressButtonLabel,
   PassImageBox,
   ProfileIconButton,
-} from './AddressBox.style';
+} from './AddressCard.style';
 
 interface AddressBoxProps {
   address?: string;
   isEVM?: boolean;
 }
 
-export const AddressBox = ({ address, isEVM }: AddressBoxProps) => {
-  const { t } = useTranslation();
+export const AddressCard = ({ address, isEVM }: AddressBoxProps) => {
+  const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
```

Comment:
Not sure the anchor is needed (I might be wrong)

Reply:
I think it is.